### PR TITLE
Lower docker image size

### PR DESCRIPTION
**Pull request by stickman33**
I suppose to lower docker image size because more than 1GB is not great at all. Tried to use alpine base image but could not to force it to work. 😶

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -1,9 +1,16 @@
...
+FROM python:3.12-slim AS builder
 
 WORKDIR /usr/src/app
 
 COPY requirements.txt ./
-RUN pip install --no-cache-dir -r requirements.txt
+
+RUN pip install --no-cache-dir --target=/usr/src/app/dependencies -r requirements.txt
+
+FROM python:3.12-slim
```
Comment (fxlrnrpt) [inline docker/Dockerfile:9]: What's the point of second FROM?

Comment (stickman33) [reply docker/Dockerfile:9]: Its to separate build and runtime environments. In runtime included only needed dependencies without building tools. It saves around 10 mb. So of course its nothing compared to the regular py image size.

**Comment by stickman33** [comment]
Added limit for video files size.

File: docker/Dockerfile (hunk 1 above)
Comment (fxlrnrpt) [reply docker/Dockerfile:9]: Oh, I see. Docker discards the previos context with a new FROM statement. Sweet!

**Pull request merged**